__tests__/models/form.test.ts

```diff
@@ -0,0 +1,113 @@
+import { Application, Resource, Form } from '@models';
```

Comment:
@ketanreliefapps for this specific model, and all models where we define schema.statics methods, we should test them

for example, we should be able to know if a form has a duplicate, meaning another form with same graphqltypename